docs_devel/docs/02.HowToBuild.md

```diff
@@ -17,13 +16,27 @@ produce Javadoc by running `gradlew javadoc`, or browse online:
 
   [https://omegat.sourceforge.io/javadoc-latest/](https://omegat.sourceforge.io/javadoc-latest/)
 
+## Prerequisite
+
+As in Dec., 2023, Here is a condition to build OmegaT binaries;
+
+1. Java Development kit 11 or later. When you want to build 
+   a linux deb/rpm package, JDK 17 or later is required.
+2. Requires `docker` or `nerdctl` command to launch containerized processes.
+   Build process for documentation and windows packages will be skipped
+   when  neither `docker` nor `nerdctl` command is not found. 
+3. JDK 21 has not been supported yet to launch Gradle build tool,
+   but it works well as an OmegaT runtime.
 
 ## Configuring Build Tasks
 
 Some build tasks, such as signed installers, require additional configuration
 via a `local.properties` file placed at the root of the source tree. See
 `local.properties.example`.
 
+When specified a Gradle property `forceSkipDocumentBuild`, the build task 
+will skip build processes to generate greeting documents, `FirstStep.html`
+in languages, and bundled manuals. You can edit `local.properties` or
```

Comment:
Was there supposed to be something else here? It seems incomplete:
"You can edit `local.properties` or"

I'm starting out on OmegaT development, trying to understand the codebase and build it myself.

Reply:
Thanks for pointing out missing sentence.
"You can edit local.properties or $HOME/.gradle./gradle.properties."

Reply:
> I'm starting out on OmegaT development, trying to understand the codebase and build it myself.

You can also check it on readthedocs website.
https://omegat.readthedocs.io/en/latest/02.HowToBuild/

Reply:
Thank you!